src/client/java/teammates/client/scripts/sql/VerifyFeedbackSessionAttributes.java

```diff
@@ -0,0 +1,55 @@
+package teammates.client.scripts.sql;
+
+import java.time.Duration;
+
+import teammates.common.util.SanitizationHelper;
+import teammates.storage.entity.FeedbackSession;
+
+/**
+ * Verification of the feedback session attributes.
+ */
+public class VerifyFeedbackSessionAttributes
+        extends VerifyNonCourseEntityAttributesBaseScript<FeedbackSession, teammates.storage.sqlentity.FeedbackSession> {
+
+    public VerifyFeedbackSessionAttributes() {
+        super(FeedbackSession.class, teammates.storage.sqlentity.FeedbackSession.class);
+    }
+
+    @Override
+    protected String generateID(teammates.storage.sqlentity.FeedbackSession sqlEntity) {
+        return FeedbackSession.generateId(sqlEntity.getName(), sqlEntity.getCourse().getId());
```

Comment:
Possible bottleneck here since you fetch course per feedback session

Reply:
I enabled sql logging and realise this line doesn't trigger a join (or select from Course table in fact). I tested by adding `sqlEntity.getCourse().getName()` and it did trigger a select from Course table if we access fields other than the FK.

This is the verification phase anyway, i think the bottleneck is eager fetching deadline extentions and feedback questions during the query phase.